client/src/components/Home.js

```diff
@@ -1,12 +1,18 @@
 import React from "react";
+import { Link } from "react-router-dom";
+import { Button } from "semantic-ui-react";
 import "../styles/Home.css";
+import "semantic-ui-css/semantic.min.css";
```

Comment:
Can this be imported in the index.js file instead? It just needs to be imported once in the project and Sharlu's PR6 will also include it in the index.js. So we can make sure we import it in the same place for everyone.

Reply:
Ok, that's right. I think index.js is more proper place for this. Thanks.